docs/content/upgrade-guides/modules/product/1-2-0.md

```diff
@@ -0,0 +1,49 @@
+---
+description: 'Actions Required for v.1.2.0'
+sidebar_custom_props:
+  iconName: 'server-stack-solid'
+---
+
+# v1.2.0
+
+Version 1.2.0 of the Product Module introduces minor breaking changes to the `ProductService`.
+
+## Overview
+
+This release contains breaking change to the methods `list` and `listAndCount` in the `ProductService` of the Product Module.
+
+Specifically, the following category filter has changed:
+```ts
```

Comment:
I think it's fine to put the before and after in the same code block

Reply:
Also it's kind of repetitive with the actions required right? I would just include the changes in the Actions Required section

Reply:
Good point, will do!